packages/seed-realm/color.gts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
My personal opinion is: The color.gts should exist inside the fields folder

Reply:
Ok, I can put it there

Reply:
Done